js/NavMainComponent.js

```diff
@@ -2,21 +2,33 @@
 
 var $ = require('jquery');
 
-function NavMainComponent(html) {
+function NavMainComponent(html, window) {
     this.$html = html;
+    this.$window = $(window);
```

Comment:
Perhaps not worth changing, but we could get both these nodes with no need for the additional argument. If we pass `html` in we could grab the window with `html.parentNode.defaultView` or if we passed in the `window` we could grab the html with `window.document.documentElement`.